index.html

```diff
@@ -216,20 +228,23 @@ <h3 class="tproj1">Profesional Art Printing Data</h3>
                 <div class="cont1">Ruby</div>
               </li>
             </ul>
-            <div id="contbut">
+            <div class="no_button">
               <button class="bigbutton1 button21" type="button">See Project</button>
             </div>
           </div>
         </div>
       </div>
     </section>
 
-    <section class="about">
-      <section name="first_about">
+    <section class="about" id="about">
+      <section id="first_about">
         <h1 id="t_about">About me</h1>
         <p id="c_about">Hello I'm a software developer! I can help you build a product , feature or website Look through
           some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to
           contact me.</p>
+        <div id="but_desk">
+          <button id="button_about" type="button">Get My Resume</button>
+        </div>
         <div id="fit_cont">
           <div>
             <button id="button_about" type="button">Get My Resume</button>
```

Comment:
- [ ] Kindly, if you check your mobile version, 🥇 is not displayed as expected, it should look like the original layout on both mobile and desktop versions, please fix it. 🈹

![image](https://user-images.githubusercontent.com/91301423/199868515-3d06bb8e-b487-4cb8-937f-1f341a5052f5.png)

<h3> Note: Please, check all sections in your mobile version</h3>